modules/administration-guide/pages/monitoring-with-prometheus-and-grafana.adoc

```diff
@@ -0,0 +1,14 @@
+:navtitle: Monitoring with Prometheus and Grafana
+:keywords: administration-guide, monitoring-with-prometheus-and-grafana
+:page-aliases: .:monitoring-with-prometheus-and-grafana
+
+:parent-context-of-monitoring-the-dev-workspace-operator: {context}
+
+[id="monitoring-with-prometheus-and-grafana"]
+= Monitoring {prod-short} with Prometheus and Grafana
+
+You can collect and view the {prod-short} metrics with a running instance of Prometheus and Grafana on the cluster.
+
+**** xref:installing-prometheus-and-grafana.adoc[]
+**** xref:monitoring-the-dev-workspace-operator.adoc[]
+**** xref:monitoring-che.adoc[]
```

Comment:
I'm sorry, I must have copied this from the nav file and forgotten to remove the extra asterisks:
```suggestion
* xref:installing-prometheus-and-grafana.adoc[]
* xref:monitoring-the-dev-workspace-operator.adoc[]
* xref:monitoring-che.adoc[]
```

Reply:
No worries, applied